dbcon/mysql/ha_mcs_sysvars.cpp

```diff
@@ -270,13 +270,28 @@ static MYSQL_THDVAR_ULONG(
     1 // block size
 );
 
-static MYSQL_THDVAR_BOOL(
+const char* mcs_use_import_for_batchinsert_values[] = {
+    "OFF",
+    "ON",
+    "ALWAYS",
+    NullS
+};
+
+static TYPELIB mcs_use_import_for_batchinsert_values_lib = {
+    array_elements(mcs_use_import_for_batchinsert_values) - 1,
+    "mcs_use_import_for_batchinsert_values",
+    mcs_use_import_for_batchinsert_values,
+    NULL
+};
+
+static MYSQL_THDVAR_ENUM(
     use_import_for_batchinsert,
-    PLUGIN_VAR_NOCMDARG,
+    PLUGIN_VAR_RQCMDARG,
```

Comment:
Are we going to set it in running MDB?

Reply:
No that's not required. PLUGIN_VAR_RQCMDARG means if we use this as a command line option, then we need to provide argument to the option. But using this variable as a command line option is not mandatory.

Reply:
I was wrong using PLUGIN_VAR_RQCMDARG. We must use PLUGIN_VAR_NOCMDOPT for all of our settings so plz start with changing it to PLUGIN_VAR_NOCMDOPT.

Reply:
PLUGIN_VAR_NOCMDOPT means the user cannot set the value of this system variable through the command line when starting up mysqld. Are we sure we don't want this behaviour?

Reply:
We are sure.